apps/nx-ghpages-e2e/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "projectType": "application",
-  "root": "apps\\nx-ghpages-e2e",
```

Comment:
These changes are not strictly necessary, but makes the project less OS dependant